docs/CONTRIBUTING.md

```diff
@@ -0,0 +1,87 @@
+# Contributing
+
+Contributions are welcome, and they are greatly appreciated! Every little bit
+helps, and credit will always be given.
+
+## Bug Reports
+
+When reporting a bug please include:
+
+*   Your operating system name and version.
+*   Any details about your local setup that might be helpful in
+    troubleshooting.
+*   Detailed steps to reproduce the bug.
+
+## Feature Requests and Feedback
+
+The best way to send feedback is to file an issue on GitHub.
+
+If you are proposing a feature:
+
+*   Explain in detail how it would work.
+*   Keep the scope as narrow as possible, to make it easier to implement.
+*   Remember that this is a community-driven, open-source project, and that
+    code contributions are welcome. :)
+
+## Development Guide
+
+### Environment Requirements
+
+Your development-system should have the following contents installed:
+
+* A `git` client
+* The `make` utility
+* An Docker-compatible container-manager (for Entrprise Linux systems, 
+  this can be satisfied with the installation of the `podman` and 
+  `podman-docker` RPMs)
+
+Note: For some distributions, the `git` and `make` tools are part of a 
+"development" package-group
+
+### Environment Setup
+
+To set up the `scap-formula` content for local modifications:
+
+1.  Fork the [scap-formula](https://github.com/plus3it/scap-formula) repository
+    (look for the `Fork` button).
+
+1.  Clone your fork locally:
+
+    ```shell
+    git clone https://github.com/<GITHUB_USERID>/scap-formula.git && cd scap-formula
+    ```
+
+1.  Add the main (plus3it) project as a remote:
+
+    ```shell
+    git remote add upstream https://github.com/plus3it/scap-formula.git
+    ```
+
+1. Ensure all remotes'/branches' contents are fully up-to-date:
+
+    ```shell
+    git fetch --all --tags -p
+    ```
+
+1.  Create a branch for local development:
+
+    ```shell
+    git checkout upstream/master -b <NAME_OF_YOUR_BUGFIX_OR_FEATURE>
+    ```
+
+### Modification of (Local) Repository Contents
+
+1.  Make desired modifications (most frequently, this will be adding, modifying
+    or removing content builder-options from the [`scripts/build_ssg_content.sh`](/scripts/build_ssg_content.sh)
+    file)
+
+1.  When desired modifications have been completed, build the content updates:
+
+    * In the root of your local repository, execute the `make` command
+    * If the `make` command fails, executing `bash ./scripts/build_in_docker.sh`
+        from the root of your local repository can be substituted
+
+    This will result in output similar to the [linked log-file](build-log_example.txt)
```

Comment:
Maybe instead of committing a new log file, just link to the one committed under the content directory?

Reply:
Looking at the `scap/content/guides/openscap/build-content.log` file, I'm seeing:
```text
Cloning into '/tmp/ComplianceAsCode/content'...
Updating files:  11% (1300/11534)^MUpdating files:  12% (1385/11534)^MUpdating files:  13% (1500/11534)^MUpdating files:  14% (1615/11534)^MUpdating files:  15% (1731/11534)^MUpdating files:  16% (1846/11534)^MUpdating files:  17% (1961/11534)^MUpdating files:  18% (2077/11534)^MUpdating files:  19% (2192/11534)^MUpdating files:  20% (2307/11534)^MUpdating files:  21% (2423/11534)^MUpdating files:  22% (2538/11534)^MUpdating files:  23% (2653/11534)^MUpdating files:  23% (2658/11534)^MUpdating files:  24% (2769/11534)^MUpdating files:  25% (2884/11534)^MUpdating files:  26% (2999/11534)^MUpdating files:  27% (3115/11534)^MUpdating files:  28% (3230/11534)^MUpdating files:  29% (3345/11534)^MUpdating files:  30% (3461/11534)^MUpdating files:  31% (3576/11534)^MUpdating files:  32% (3691/11534)^MUpdating files:  33% (3807/11534)^MUpdating files:  33% (3912/11534)^MUpdating files:  34% (3922/11534)^MUpdating files:  35% (4037/11534)^MUpdating files:  36% (4153/11534)^MUpdating files:  37% (4268/11534)^MUpdating files:  38% (4383/11534)^MUpdating files:  39% (4499/11534)^MUpdating files:  40% (4614/11534)^MUpdating files:  41% (4729/11534)^MUpdating files:  42% (4845/11534)^MUpdating files:  43% (4960/11534)^MUpdating files:  44% (5075/11534)^MUpdating files:  45% (5191/11534)^MUpdating files:  46% (5306/11534)^MUpdating files:  46% (5320/11534)^MUpdating files:  47% (5421/11534)^MUpdating files:  48% (5537/11534)^MUpdating files:  49% (5652/11534)^MUpdating files:  50% (5767/11534)^MUpdating files:  51% (5883/11534)^MUpdating files:  52% (5998/11534)^MUpdating files:  53% (6114/11534)^MUpdating files:  54% (6229/11534)^MUpdating files:  55% (6344/11534)^MUpdating files:  56% (6460/11534)^MUpdating files:  57% (6575/11534)^MUpdating files:  58% (6690/11534)^MUpdating files:  58% (6782/11534)^MUpdating files:  59% (6806/11534)^MUpdating files:  60% (6921/11534)^MUpdating files:  61% (7036/11534)^MUpdating files:  62% (7152/11534)^MUpdating files:  63% (7267/11534)^MUpdating files:  64% (7382/11534)^MUpdating files:  65% (7498/11534)^MUpdating files:  66% (7613/11534)^MUpdating files:  67% (7728/11534)^MUpdating files:  68% (7844/11534)^MUpdating files:  69% (7959/11534)^MUpdating files:  70% (8074/11534)^MUpdating files:  70% (8081/11534)^MUpdating files:  71% (8190/11534)^MUpdating files:  72% (8305/11534)^MUpdating files:  73% (8420/11534)^MUpdating files:  74% (8536/11534)^MUpdating files:  75% (8651/11534)^MUpdating files:  76% (8766/11534)^MUpdating files:  77% (8882/11534)^MUpdating files:  78% (8997/11534)^MUpdating files:  79% (9112/11534)^MUpdating files:  80% (9228/11534)^MUpdating files:  81% (9343/11534)^MUpdating files:  82% (9458/11534)^MUpdating files:  83% (9574/11534)^MUpdating files:  83% (9647/11534)^MUpdating files:  84% (9689/11534)^MUpdating files:  85% (9804/11534)^MUpdating files:  86% (9920/11534)^MUpdating files:  87% (10035/11534)^MUpdating files:  88% (10150/11534)^MUpdating files:  89% (10266/11534)^MUpdating files:  90% (10381/11534)^MUpdating files:  91% (10496/11534)^MUpdating files:  92% (10612/11534)^MUpdating files:  92% (10720/11534)^MUpdating files:  93% (10727/11534)^MUpdating files:  94% (10842/11534)^MUpdating files:  95% (10958/11534)^MUpdating files:  96% (11073/11534)^MUpdating files:  97% (11188/11534)^MUpdating files:  98% (11304/11534)^MUpdating files:  99% (11419/11534)^MUpdating files: 100% (11534/11534)^MUpdating files: 100% (11534/11534), done.
```
Plus, I thought that maybe showing the Docker-related activities might be helpful?

Reply:
I think I'd rather just update the way the build-content.log is captured so it gets everything, than commit some 95% the same output that's going to go stale. Should just be a matter of `tee`'ing the command in the make target?

Reply:
Currently, the `scripts/build_ssg_content.sh` script is [already `tee`ing output](https://github.com/plus3it/scap-formula/blob/master/scripts/build_ssg_content.sh#L29-L30):
```bash
mkdir -p "$DIST_DIR"
exec > >(tee "${DIST_DIR}/build-content.log") 2>&1
```
It's just that it's missing the Docker-emitted events (and creating the long-assed `updating files` line rather than snarfing only the success status of that `git clone` operation).

Probably easier just to omit it all rather than try to "coddle" contributor executing the code (i.e., just point at the "as is" log-file rather than trying to spiff-up the already-generated file's contents.